File: ProblemSolving/HackerRank_Tree_Problems/level_order_traversal.py
def levelOrder(root):
    #Write your code here
    queue = []
    queue.append(root)
    while len(queue):
        root = queue.pop(0)
        print(root.info,end=" ")
        if root.left:
            queue.append(root.left)
        if root.right:
            queue.append(root.right)

File: Problem Solving/HackerRank_Tree_Problems/level_order_traversal.cpp
void levelOrder(Node * root) {
        static queue <Node*> store;
        Node *temp=NULL;
        store.push(root);
        while(!store.empty()){
            temp=store.front();
            cout<<temp->data<<" ";
            store.pop();
            if(temp->left)
              store.push(temp->left);
            if(temp->right)
              store.push(temp->right);
        }
    }
